corehq/apps/reports/commtrack/standard.py

```diff
@@ -109,19 +109,19 @@ def shared_pagination_GET_params(self):
             {'name': 'program', 'value': self.request.GET.get('program')},
         ]
 
-    @memoized
     @property
+    @memoized
```

Comment:
You can also use `django.utils.functional.cached_property` for these - I also always mess up the order :tongue:

Reply:
*legends* :)